src/hint_cli.py

```diff
@@ -98,13 +98,34 @@ def remove_config(path):
         os.unlink(p)
 
 
+def verify_data_loss(action, args, cfg):
+    if action == "stop" and args["remove_volumes"]:
+        if cfg.protect_data:
+            raise Exception("Cannot remove volumes with this configuration")
+        else:
+            print("""WARNING! PROBABLE IRREVERSIBLE DATA LOSS!
+
+You are about to delete the data volumes. This action cannot be undone
+and will result in the irreversible loss of *all* data associated with
+the application. This includes the database, the uploaded files, the
+keypairs used to sign login requests, etc.""")
+
+            if not prompt_yes_no():
+                raise Exception("Not continuing")
+
+
+def prompt_yes_no(get_input=input):
+    return get_input("Continue? [yes/no] ") == "yes"
```

Comment:
Could make this accept "y/n" if you want to save keystrokes?

Reply:
Hah - I think is not too much of a hardship :)